src/run/runner/wall_time/perf.rs

```diff
@@ -0,0 +1,63 @@
+use crate::run::runner::helpers::setup::run_with_sudo;
+use codspeed::fifo::FifoIpc;
+use std::io::{Read, Write};
+
+pub const PERF_CTL_FIFO: &str = "/tmp/codspeed_perf.ctl.fifo";
```

Comment:
this should be an unique tmp dir for this runner instance imo, so that we don't have undefined behavior if people decide to run two walltime concurently (which is stupid, I give you that, but hey)